src/testing/utils.tsx

```diff
@@ -218,6 +221,10 @@ export const getUrlParam: URLSearchParams["get"] = (param: string) =>
 
 // Complete initial test state with all data loaded and no errors
 export const getTestState = (): RootState => {
+  const config = configFactory({
+    name: ConfigNames.SESSION_LENGTH,
+    value: 1209600,
```

Comment:
It might be good to either use a named variable for this value or add a comment explaining this number.

Reply:
I think I'll do the latter, 1209600 seconds is 14 days, the default value